.cargo/config.toml

```diff
@@ -0,0 +1,23 @@
+[target.'cfg(any(test, not(test)))']
```

Comment:
I was already wondering last time I saw this, but put it off as example.
Now that what seems like a noop/`if true` kind of statement appears again, I wonder if it has a particular effect that I am unaware of?

Reply:
there is no `cfg(true)` in Rust so the only way to do it is to say `A or not(A)`, which this is doing.

Theorically I could do `cfg(not(this_cfg_does_not_exist))` but then someone could pass `--cfg this_cfg_does_not_exist` to cargo and the flags wouldn't be picked up: this is not possible when the condition is the one I used

Reply:
And removing that line should work the same, right? (*even though I have a feeling that it's significant*)

Reply:
No, it wouldn't work at all I think, I could also have used `[build]` but `rustflags` needs to be in either a `target.<...>` or `build` table

Reply:
I'll check with `[build]`, it'll be simpler, I shouldn't have copied my conditions from another project